test_gpios.py

```diff
@@ -101,14 +107,28 @@ async def test_interrupts_and_pwm(self, pwm):
 
         starting_count = await interrupt.value()
         await asyncio.sleep(DURATION)
+        should_stop.set()
+        await counter_task
+
         await pwm_pin.set(False) # Turn the output off again
+        print("data from tick stream:")
+        for tick in ticks:
+            print(tick)
```

Comment:
I'm not wild about `print`, but it seemed easier than inserting logging in here. :shrug: